ckanext/msl_ckan/schemas/datasets/cd2_data.collection.yml

```diff
@@ -49,20 +58,30 @@ dataset_fields:
     label: End date of wave collection
   - field_name: dc_wave_coverage # note the prefix and see the model: this is actually a field that applies to a single measurement
     label: Coverage percentage of the measurement for this wave
-
+  # DH: added Physical data product
+  - field_name: wave_physical_data_product
+    label: Documents the physical structure of the data produced by the study unit, e.g., digital files, biomedical samples, etc.
+    
   # - field_name: wave_creators
     #   removed for wave level because these are included with the data collection itself
     # - field_name: wave_contributors
     #   removed for wave level because these are included with the data collection itself
 
   # Is this actually necessary? Don't think so...:
-  # - field_name: wave_funding # can be distinct for different waves
-    #   label: Funding information
-    #   repeating_subfields:
-    #   - field_name: wave_funder
-    #     label: Funder name
-    #   - field_name: wave_grant_number
-    #     label:  Grant number
+  # DH: I think it is worth adding here, you never know if a specific wave is funded.
+   - field_name: wave_funding # can be distinct for different waves
+       label: Funding for this wave
+       repeating_subfields:
+       - field_name: wave_funder
+         label: Name of funder
+       - field_name: wave_grant_number
+         label:  Grant number
+
+# DH: Fields also included in metadata model:
+#- field_name: wave_language
+#  label: Language of the wave
+#- field_name: wave_international_identifier
+#  label: Wave international identifier # e.g. DOI
```

Comment:
See previous notes on language and international identifier. If we decide to enable these fields, the same adjustments should be able as on the other instances of the language and international identifier fields.